packages/reports/app/styles/navi-reports/components/navi-modal.less

```diff
@@ -54,6 +54,10 @@
     line-height: inherit;
   }
 
+  &__denali-input {
+    width: 580px !important;
```

Comment:
Is this being used?

Reply:
Not anymore. I added this for the modal inputs